src/rust/Cargo.toml

```diff
@@ -21,8 +21,14 @@ members = [
     "bitbox02-noise",
     "bitbox02",
     "bitbox02-sys",
+    "erc20_params",
 ]
 
+# Needed if edition 2018 and 2021 are mixed in the workspace crates, to fix
+# a cargo warning.
+# When all are switched to 2021, this can be removed.
+resolver = "2"
+
```

Comment:
Now part of https://github.com/digitalbitbox/bitbox02-firmware/pull/1216, which should be merged first, then this can be rebased.

Reply:
I tested and reviewed that pr first, it's ready to merge.